src/db/sqlite/lrsql/sqlite/sql/ddl.sql

```diff
@@ -329,7 +329,7 @@ SET sql = 'CREATE TABLE credential_to_scope (
 )'
 WHERE type = 'table' AND name = 'credential_to_scope'
 
-/* Migration 2022-05-08-00 - Add timestamp to xapi_statement */
+/* Migration 2023-05-08-00 - Add timestamp to xapi_statement */
```

Comment:
It's the "it's still the previous year!" feeling isn't it?

Reply:
yep